ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/af-commons/v1/fieldTemplates/characterCounter.html

```diff
@@ -0,0 +1,6 @@
+<template data-sly-template.characterCounter="${@ componentId, showCharCounter, maxLength, bemBlock}">
+  <div data-sly-test="${showCharCounter}" class="${bemBlock}__charcounter">
```

Comment:
wrapper is missing. You will need to add aria label for accessibility as well.

Reply:
@sakshi-arora1  I have added aria label, but didn't understand "adding wrapper"?